phoenix-core/src/main/java/org/apache/phoenix/hbase/index/IndexRegionObserver.java

```diff
@@ -506,7 +506,7 @@ private void populatePendingRows(BatchMutateContext context, long now) {
       return mutations;
   }
 
-  public void removeEmptyColumn(Mutation m, byte[] emptyCF, byte[] emptyCQ) {
+  public static void removeEmptyColumn(Mutation m, byte[] emptyCF, byte[] emptyCQ) {
```

Comment:
@gokceni, @kadirozde, IMPO, we should move the following list from IndexRegionObserver to an MutationUtil class in org.apache.phoenix.hbase.index.util or org.apache.phoenix.util:

IndexRegionObserver.UNVERIFIED_BYTES
IndexRegionObserver.VERIFIED_BYTES
IndexRegionObserver.getMaxTimestamp(...)
IndexRegionObserver.removeEmptyColumn(...)